app/services/concerns/payloadable.rb

```diff
@@ -1,13 +1,21 @@
 # frozen_string_literal: true
 
 module Payloadable
+  # @param [ActiveModelSerializers::Model] record
+  # @param [ActiveModelSerializers::Serializer] serializer
+  # @param [Hash] options
+  # @option options [Account] :signer
+  # @option options [String] :sign_with
+  # @option options [Boolean] :always_sign
```

Comment:
Maybe describe what `always_sign` actually does. “Sign even when `AUTHORIZED_FETCH` is set” or something like that